mem-13-Q.xml

```diff
@@ -5791,17 +5893,17 @@ The slang meaning of {chong:v:2,slang}, when applied to a person, is equivalent
       <column name="definition_de">Kruge</column>
       <column name="definition_fa">Kruge [AUTOTRANSLATED]</column>
       <column name="definition_sv">Kruge</column>
-      <column name="definition_ru"></column>
+      <column name="definition_ru">Крюге</column>
       <column name="definition_zh_HK">「克魯格」</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also"></column>
-      <column name="notes"></column>
+      <column name="notes">The transliteration comes from the subtitles of {Star Trek III:src}.[2]</column>
```

Comment:
This change is not correct. This note belongs under "notes_zh_HK" and does not need to be translated into Russian, as it is a reference to the Chinese transliteration.

Reply:
That is, unless your transliteration also comes from the subtitles of Star Trek III, in which case, the same note would be applicable.

Reply:
fixed